braph2genesis/graphs/_OrdMlWD.gen.m

```diff
@@ -0,0 +1,281 @@
+    'YSLIDERSHOW', true, ...
+    'YSLIDERLABELS', g.getCallback('ALAYERLABELS'), ...
+    'YSLIDERWIDTH', s(5), ...
+    'ROWNAME', g.getCallback('ANODELABELS'), ...
+    'COLUMNNAME', g.getCallback('ANODELABELS'), ...
+    varargin{:});
+
+%%% ¡prop!
+PARTITIONS (result, rvector) returns the number of layers in the partitions of the graph.
+%%%% ¡calculate!
+value = ones(1, g.get('LAYERNUMBER'));
+
+%%% ¡prop!
+ALAYERLABELS (query, stringlist) returns the layer labels to be used by the slider.
+%%%% ¡calculate!
+alayerlabels = g.get('LAYERLABELS');
+if isempty(alayerlabels) && ~isa(g.getr('A'), 'NoValue') % ensures that it's not unecessarily calculated
+    alayerlabels = cellfun(@num2str, num2cell([1:1:g.get('LAYERNUMBER')]), 'uniformoutput', false);
+end
+value = alayerlabels;
+
+%%% ¡prop!
+COMPATIBLE_MEASURES (constant, stringlist) is the list of compatible measures.
+%%%% ¡default!
+getCompatibleMeasures('OrdMlWD')
+
+%% ¡props!
+
+%%% ¡prop!
+B (data, cell) is the input cell containing the multilayer adjacency matrices.
+%%%% ¡default!
+{[] []; [] []}
+%%%% ¡gui!
+pr = PanelPropCell('EL', g, 'PROP', OrdMlWD.B, ...
+    'TABLE_HEIGHT', s(40), ...
+    'XSLIDERSHOW', true, ...
+    'XSLIDERLABELS', g.get('LAYERLABELS'), ...
+    'XSLIDERHEIGHT', s(3.5), ...
+    'YSLIDERSHOW', false, ...
+    'ROWNAME', g.getCallback('ANODELABELS'), ...
+    'COLUMNNAME', g.getCallback('ANODELABELS'), ...
+    varargin{:});
+
+
+%%% ¡prop!
+SEMIPOSITIVIZE_RULE (parameter, option) determines how to remove the negative edges.
+%%%% ¡settings!
+{'zero', 'absolute'}
+
+%%% ¡prop!
+STANDARDIZE_RULE (parameter, option) determines how to normalize the weights between 0 and 1.
+%%%% ¡settings!
+{'threshold' 'range'}
+
+%% ¡tests!
+
+%%% ¡excluded_props!
+[OrdMlWD.PFGA OrdMlWD.PFGH]
+
+%%% ¡test!
+%%%% ¡name!
+Constructor - Full
+%%%% ¡probability!
+.01
+%%%% ¡code!
+B1 = rand(randi(10));
+B2 = rand(randi(10));
+B3 = rand(randi(10));
+B12 = rand(size(B1,1),size(B2,2));
+B13 = rand(size(B1,1),size(B3,2));
+B23 = rand(size(B2,1),size(B3,2));
+B21 = rand(size(B2,1),size(B1,2));
+B31 = rand(size(B3,1),size(B1,2));
+B32 = rand(size(B3,1),size(B2,2));
+B = {
+    B1                           B12                            B13
+    B21                          B2                             B23
+    B31                          B32                            B3
+    };
+g = OrdMlWD('B', B);
+g.get('A_CHECK')
+A1 = standardize(semipositivize(dediagonalize(B1)));
+A2 = standardize(semipositivize(dediagonalize(B2)));
+A3 = standardize(semipositivize(dediagonalize(B3)));
+A12 = standardize(semipositivize(B12));
+A13 = standardize(semipositivize(B13));
+A23 = standardize(semipositivize(B23));
+A21 = standardize(semipositivize(B21));
+A31 = standardize(semipositivize(B31));
+A32 = standardize(semipositivize(B32));
+B{1,1} = A1;
+B{2,2} = A2;
+B{3,3} = A3;
+B{1,2} = A12;
+B{1,3} = A13;
+B{2,3} = A23;
+B{2,1} = A21;
+B{3,1} = A31;
+B{3,2} = A32;
+A = B
```

Comment:
leave a space after the comman, e.g. B{3, 2} = A32;
And spread this changes to all the scripts